common/common.cpp

```diff
@@ -1942,8 +1940,21 @@ static bool llama_download_file(CURL * curl, const std::string & url, const std:
                 fprintf(stderr, "%s: error reading metadata file %s: %s\n", __func__, metadata_path.c_str(), e.what());
                 return false;
             }
-
+        } else {
+            // Check for legacy ${path_model}.etag & ${path_model}.lastModified files (note: both streams autoclosed upon exiting this block).
```

Comment:
Thanks, but this is probably overkill, our users are used to breaking changes ;) let's start from fresh download again and clean code

Reply:
Fair enough 😅

Added a caveat to the PR description + a Python snippet for manual migration to the new `.json` world.